broker/src/test/java/io/zeebe/broker/exporter/stream/ExporterDirectorTest.java

```diff
@@ -84,6 +84,134 @@ private void startExporterDirector(final List<ExporterDescriptor> exporterDescri
     rule.startExporterDirector(exporterDescriptors);
   }
 
+  @Test
+  public void shouldUpdatePositionWhenInitialRecordsAreSkipped() {
+    // given
+    final ControlledTestExporter tailingExporter = exporters.get(0);
```

Comment:
True - it highlights that the director is difficult to test, since now we're relying that the order in which the record is exported will be the same, which is implementation details. Let's go with that now, but in the not too distant future we probably need to refactor the director to make it more testable.